axum/src/extract/rejection.rs

```diff
@@ -111,7 +110,6 @@ impl FailedToDeserializeQueryString {
     {
         FailedToDeserializeQueryString {
             error: Error::new(error),
-            type_name: std::any::type_name::<T>(),
```

Comment:
I think we should remove the `T` type param as well since it's not used anymore.